root/commons/src/main/java/at/fhv/itb5c/commons/enums/Gender.java

```diff
@@ -1,4 +1,4 @@
-package at.fhv.itb5c.model.entity;
+package at.fhv.itb5c.commons.enums;
 
 public enum Gender {
```

Comment:
As these enums are now in the *commons* package: Maybe we could use them for converting into basic data types (-->String)? E.g.
```java
public static Gender getGender(String name) {
  switch(name) // etc.
  return Gender.Female; // etc.
}
// and a method for conversion to String as well
```